src/hooks/__internal__/useCharacterCount/useCharacterCount.test.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import { act } from "react-dom/test-utils";
+import useCharacterCount from ".";
+
+interface TestComponentProps {
+  value: string;
+  characterLimit?: number;
+}
+
+const TestComponent = ({ value, characterLimit }: TestComponentProps) => {
+  const [characterCount] = useCharacterCount(value, characterLimit);
+
+  return <div>{characterCount}</div>;
+};
+
+const MOCK_VALUE = "test string";
+
+test.each([5, 10, 15])(
```

Comment:
**suggestion**: what do you think of splitting this test, so we assert being over and under the limit separately? That way we can avoid using conditionals in our tests.